### PR TITLE
point k8s at latest

### PR DESCRIPTION
**Pull request by jacksonrnewhouse**
Also sets the K8S_IMAGE variable for the Helm chart.

### CODE REVIEW

File: k8s/arroyo/values.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ api:
   image:
     repository: ghcr.io/arroyosystems/arroyo-services
     pullPolicy: IfNotPresent
-    tag: "amd64"
+    tag: "latest"
```
Comment (mwylde) [inline k8s/arroyo/values.yaml:18]: is this tag the one that will pull the correct image for your arch?

Comment (jacksonrnewhouse) [reply k8s/arroyo/values.yaml:18]: We're not currently automatically pushing to latest. I think we can either do it manually when we cut a release or setup a separate task. My inclination is to not build on every merge. Maybe every tag like `[0-9]+.[0-9]+.[0-9]`, but then publishing an old tag would roll back.

Comment (mwylde) [reply k8s/arroyo/values.yaml:18]: I actually was asking what's the difference between `multi-arch` and `latest`. Is `latest` pointing to a particular architecture, or is it the multi-arch manifest?

Comment (jacksonrnewhouse) [reply k8s/arroyo/values.yaml:18]: I think we can publish a latest manifest. Alternately, we could keep it pinned to an explicit version.

**Review by mwylde — APPROVED** [review]

**Pull request closed without merging**